packages/jbrowse-cli/src/commands/create.test.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * @jest-environment node
+ */
+
+import fs from 'fs'
+import * as path from 'path'
+import nock from 'nock'
+import del from 'del'
+import { setup } from '../testUtil'
+
+const fsPromises = fs.promises
+const testDir = path.join(
+  __dirname,
+  '..',
+  '..',
+  'test',
+  'data',
+  'createTestDir',
+)
+
+nock('https://s3.amazonaws.com')
```

Comment:
This implies that this is a resource that we will maintain indefinitely? Has this been discussed?

Reply:
Thanks for gettin this checked out :)